### PR TITLE
Added some constants and their uncertainty to TMath

### PR DESCRIPTION
**Pull request by AdvaitDhingra**
I added some constants to TMath.h . They are widely used constants, so having them makes sense. I also added the uncertainties.

### CODE REVIEW

**Comment by AdvaitDhingra** [comment]
@lmoneta is this PR meaningful enough to be merged? If not, I can find more useful constants to add.

**Comment by linev** [comment]
Please rebase your branch

**Comment by AdvaitDhingra** [comment]
Oh no I did it again :/

**Comment by AdvaitDhingra** [comment]
How do I only keep my commits?

**Comment by eguiraud** [comment]
`git rebase -i upstream/master ` and edit the history at your leisure (you should not use your master branch but a dedicated feature branch to work on something specific).

**Comment by AdvaitDhingra** [comment]
> `git rebase -i upstream/master ` and edit the history at your leisure (you should not use your master branch but a dedicated feature branch to work on something specific).

Ah, thanks. I tried cherry-pick but that wasnt working

**Comment by AdvaitDhingra** [comment]
Nevermind, it turns out I was cherry-picking on the wrong branch lol

**Comment by AdvaitDhingra** [comment]
Is this PR ok to be merged?

**Review by lmoneta — APPROVED** [review]

**Comment by lmoneta** [comment]
@phsft-bot build!

File: math/mathcore/inc/TMath.h [hunk 1]
```diff
@@ -271,6 +285,20 @@ constexpr Double_t KUncertainty()
    return 0.0;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Coulomb Constant in \f$ kgm^3s^{-2}C^{-2} \f$
+constexpr Ke()
```
Comment (lmoneta) [inline math/mathcore/inc/TMath.h:290]: Please add here the declaration (e.g. Double_t)

**Review by lmoneta — CHANGES_REQUESTED** [review]

File: math/mathcore/inc/TMath.h (hunk 1 above)
Comment (AdvaitDhingra) [reply math/mathcore/inc/TMath.h:290]: Ah, ok.

**Comment by AdvaitDhingra** [comment]
That was a pretty basic mistake, sorry 😅

**Comment by AdvaitDhingra** [comment]
Can the build be aborted and restarted ?

**Comment by AdvaitDhingra** [comment]
Can we restart the build with @phsft-bot ?

**Pull request closed without merging**